script.js

```diff
@@ -1,4 +1,98 @@
+var startingPoint = 0;
+var losingPoint = 0;
+var drawPoint = 0;
+var userName = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var userName = input;
```

Comment:
implement use of game modes to switch between asking for username and playing the game